### PR TITLE
fix: removed email icon from breadcrumb and fixed mobile alignment

### PR DESCRIPTION
**Pull request by alessandrogelmi**
## Short description
Removed icons from breadcrumb of notification detail. Fixed also mobile alignment of breadcrumb

## List of changes proposed in this pull request
- Removed icon from breadcrum
- Fixed mobile alignment

## How to test
Go to notification detail of PA, PF and PG and breadcrumb shouldn't have the email icon. On mobile it should be inline

### CODE REVIEW

**Comment by alessandrogelmi** [comment]
[PN-6086](https://pagopa.atlassian.net/browse/PN-6086)
[PN-6622](https://pagopa.atlassian.net/browse/PN-6622)
[PN-6731](https://pagopa.atlassian.net/browse/PN-6731)

File: packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx [hunk 1]
```diff
@@ -340,12 +339,7 @@ const NotificationDetail = () => {
   const properBreadcrumb = (
     <PnBreadcrumb
       linkRoute={routes.DASHBOARD}
-      linkLabel={
-        <Fragment>
-          <EmailIcon sx={{ mr: 0.5 }} />
-          {t('detail.breadcrumb-root', { ns: 'notifiche' })}
-        </Fragment>
-      }
+      linkLabel={<Fragment>{t('detail.breadcrumb-root', { ns: 'notifiche' })}</Fragment>}
```
Comment (AndreaCimini90) [inline packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx:342]: Remove fragment

File: packages/pn-personafisica-webapp/src/pages/NotificationDetail.page.tsx [hunk 2]
```diff
@@ -267,12 +266,7 @@ const NotificationDetail = () => {
       <PnBreadcrumb
         showBackAction={!fromQrCode}
         linkRoute={backRoute}
-        linkLabel={
-          <Fragment>
-            <EmailIcon sx={{ mr: 0.5 }} />
-            {t('detail.breadcrumb-root', { ns: 'notifiche' })}
-          </Fragment>
-        }
+        linkLabel={<Fragment>{t('detail.breadcrumb-root', { ns: 'notifiche' })}</Fragment>}
```
Comment (AndreaCimini90) [inline packages/pn-personafisica-webapp/src/pages/NotificationDetail.page.tsx:269]: Same as before

File: packages/pn-personagiuridica-webapp/src/pages/NotificationDetail.page.tsx [hunk 3]
```diff
@@ -267,12 +266,7 @@ const NotificationDetail = () => {
       <PnBreadcrumb
         showBackAction={!fromQrCode}
         linkRoute={backRoute}
-        linkLabel={
-          <Fragment>
-            <EmailIcon sx={{ mr: 0.5 }} />
-            {t('detail.breadcrumb-root', { ns: 'notifiche' })}
-          </Fragment>
-        }
+        linkLabel={<Fragment>{t('detail.breadcrumb-root', { ns: 'notifiche' })}</Fragment>}
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/pages/NotificationDetail.page.tsx:269]: Same as before

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**